web/run.py

```diff
@@ -0,0 +1,25 @@
+"""
```

Comment:
So is the website now supposed to be launched with `python web/run.py` instead of `python web/web.py`? If so, can you update the README?